src/mlpack/methods/preprocess/preprocess_one_hot_encoding_main.cpp

```diff
@@ -0,0 +1,84 @@
+/**
+ * @file methods/preprocess/preprocess_one_hot_encoding_main.cpp
+ * @author Jeffin Sam
+ *
+ * A binding to do one-hot encoding on features from a dataset.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <mlpack/prereqs.hpp>
+#include <mlpack/core/util/io.hpp>
+#include <mlpack/core/util/mlpack_main.hpp>
+#include <mlpack/core.hpp>
+#include <mlpack/core/data/one_hot_encoding.hpp>
+
+// Program Name.
+BINDING_NAME("One Hot Encoding");
+
+// Short description.
+BINDING_SHORT_DESC("A utility to do one hot encoding on features of dataset.");
```

Comment:
```suggestion
BINDING_SHORT_DESC("A utility to do one-hot encoding on features of dataset.");
```

I think that is the common term?